examples/resources/conf/migrate_dataset.config.yaml

```diff
@@ -0,0 +1,19 @@
+source_migration_instance:
+    host: $DEV_HOST
+    protocol: "http"
+    port: "9100"
+    username: $TAMR_USERNAME
+    password: $TAMR_PASSWORD
+
+target_migration_instance:
+    host: $PROD_HOST
+    protocol: "http"
+    port: "9100"
+    username: $TAMR_USERNAME
+    password: $TAMR_PASSWORD
+
+logging_dir: $TAMR_PROJECT_LOGGING_DIR # Example: "/home/users/jane/my-project/logs"
+
+datasets:
+    - dataset_name: "test_dataset"
+      dataset_primary_key: "unique_id"
```

Comment:
Can you add a newline at the end here?

Reply:
Looks like a tab got included here, can you clear that so it's just an empty newline with no whitespace?